src/hotspot/share/classfile/classLoaderDataGraph.cpp

```diff
@@ -268,14 +269,6 @@ inline void assert_is_safepoint_or_gc() {
          "Must be called by safepoint or GC");
 }
 
-void ClassLoaderDataGraph::cld_unloading_do(CLDClosure* cl) {
```

Comment:
Thanks for noticing that we're not using this anymore and cleaning it up.  JFR used to use this.